frontend/src/bundles/common/helpers/calculate-age/calculate-age.ts

```diff
@@ -0,0 +1,21 @@
+const calculateAge = (dateOfBirth: string | null): number | null => {
```

Comment:
file name should have helpers suffix
Also I think could be simplified by using date-fns